lib/Payplug/Payplug.php

```diff
@@ -17,19 +19,53 @@ class Payplug
      */
     private $_token;
 
+    /**
+     * @var string The API Version.
+     */
+    private $_api_version;
+
     /**
      * Constructor for a authentication.
      *
-     * @param   string  $token  The token
+     * @param string $token The token
+     * @param string $api_version The API Version
      *
      * @throws Exception\ConfigurationException when token is not set
      */
-    public function __construct($token)
+    public function __construct($token, $api_version)
```

Comment:
Si je dis pas de bêtise on préférera `$apiVersion`

Reply:
Noté